Sparkle/SUUpdater.m

```diff
@@ -70,14 +70,15 @@ @implementation SUUpdater
 static NSString *const SUUpdaterDefaultsObservationContext = @"SUUpdaterDefaultsObservationContext";
 
 // Debug is not defined in released builds and pedantic mode can enable -Wundef
-#if defined(DEBUG) && DEBUG
-+ (void)load
-{
-    // Debug builds have different configurations for update check intervals
-    // We're using NSLog instead of SULog here because we don't want to start Sparkle's logger here
-    NSLog(@"WARNING: This is running a Debug build of Sparkle; don't use this in production!");
-}
-#endif
+// 45900: XC14 doesn't like this at all, so killing this to silence the warning.
```

Comment:
Comment for clarity. It seems to be our own addition for reasons that probably don't even make sense and won't for long when we update Sparkle anyway. The trouble is the use of `+load` was triggering a warning about the use of this class method slowing down launch times. So it's gone but not forgotten (see what I did there? commented? eh? EH??).